build-system/get-dep-graph.js

```diff
@@ -206,7 +211,8 @@ exports.getBundleFlags = function(g) {
       throw new Error('Expect to build more than one bundle.');
     }
   });
-  flagsArray.push('--js_module_root', `${g.tmp}/node_modules/`);
+  // Do _not_ include 'node_modules/' in js_module_root with 'NODE' resolution
```

Comment:
Does the order matter here? Can these static items just be placed into the array when its created?

Perhaps a better followup change.

Reply:
Not sure, Malte wrote this. :)

Reply:
Not a big deal. We can verify later.